MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
+        /// </summary>
+        public class Random
+        {
+            #region Static methods
+
+            /// <summary>
+            /// Utility method for generating a seed for the random number generator.
+            /// </summary>
+            [CLSCompliant(false)]
+            public static ulong GenerateSeed()
+            {
+                byte[] guid = Guid.NewGuid().ToByteArray();
+                return (BitConverter.ToUInt64(guid, 0) ^ BitConverter.ToUInt64(guid, 7));
```

Comment:
Just want to add note that for Guid, `ToByteArray()` and `ToString()` return different values.